tests/Client/response_server.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Take the HTTP request and build the desiderata response
```

Comment:
"desiderata" :)

Reply:
Yes, it's in Latin. Maybe it's better to use "desired response" :)

Reply:
I assumed it was something that slipped in italian, heh. It's totally fine by me, no need to change anything.